main.go

```diff
@@ -412,6 +421,20 @@ func getCACert() (string, error) {
 	return cs.Data["certificate"].(string), nil
 }
 
+func fetchOvpnKey(fqdn string) (string, error) {
+	path := strings.Join([]string{"secret", "data", strings.Trim(cfg.OvpnKey, "/")}, "/")
```

Comment:
Please do not assume the key is stored in `secret/data/...`. The KV store can be mounted at any name so the user needs to supply the whole path. Also the user most likely will not use `<mount-point>/data/<keyname>`.